.travis.yml

```diff
@@ -73,7 +79,10 @@ install:
   - pip install -U pip setuptools
   - pip install -U coverage coveralls
   - pip install -U -e .[test]
-  - if [[ $PURE_PYTHON ]]; then pip install -U -e .[zodb,zcml]; fi
+  - |
+    if [[ "$PURE_PYTHON" == "0" || ${TRAVIS_PYTHON_VERSION} == pypy* ]]; then
+      pip install -U -e .[zodb,zcml];
+    fi
 
```

Comment:
This reverses the logic?  Previously the extra pip install was run only for PURE_PYTHON=1.

(I don't understand what the purpose of it was in any case.  Enable additional tests to run that depend on zodb/zcml extras and would be skipped otherwise?  Why not do that always -- are we trying to speed up Travis runs or what?  A comment would be helpful.)

Reply:
In the distant past, we didn't test the ZODB integration at all, and it turned out there were bugs in the combination of contained proxies, persistence, and pickling, but only in the pure-python case. So we started to test the ZODB extra only in that case. I made sure to preserve that with PyPy, which is where it matters most.

I had assumed that there was significant coverage differences if ZODB wasn't available, but when I checked just now, it looks like there really aren't. There are no big blocks of code that do anything differently if ZODB isn't available that need to be tested. 

So I think it makes sense just to include those two extras in all test environments. I'll make that change.